File: exercises/RB101_small_problems/easy_1/madlibs.rb

puts "Enter a noun:"
noun = gets.chomp

puts "Enter a verb:"
verb = gets.chomp

puts "Enter an adjective:"
adjective = gets.chomp

puts "Enter an adverb:"
adverb = gets.chomp

sentence_1 = "Do you #{verb} your #{adjective} #{noun} #{adverb}? That's hilarious!"
sentence_2 = "The #{adjective} #{noun} #{verb}s #{adverb} over the lazy dog."
sentence_3 = "The #{noun} #{adverb} #{verb}s up #{adjective} Joe's turtle."

puts [sentence_1, sentence_2, sentence_3].sample

File: exercises/JS210_Small_Problems/Easy_2/madlibs.js
const rlSync = require('readline-sync');

function madlib() {
  let noun = rlSync.question("Enter a noun: ");
  let verb = rlSync.question("Enter a verb: ");
  let adjective = rlSync.question("Enter an adjective: ");
  let adverb = rlSync.question("Enter an adverb: ");

  console.log(`Do you ${verb} your ${adjective} ${noun} ${adverb}? That's hilarious!`);
}

madlib();

// Enter a noun: dog
// Enter a verb: walk
// Enter an adjective: blue
// Enter an adverb: quickly

// // console output
// Do you walk your blue dog quickly? That's hilarious!